packages/core/addon/components/number-format-selector.hbs

```diff
@@ -0,0 +1,48 @@
+{{!-- Copyright 2021, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+<div class="number-format-selector" ...attributes>
+  <DenaliInputGroup @label="Format Type" @isStacked= {{true}} class="number-format-selector__section-header">
+    <DenaliRadio
+      @onChanged={{queue @onUpdateFormat (fn (mut this.format))}}
+      as |Radio|
+    >
+      {{#each this.predefinedFormats as |predefinedFormat|}}
+        <div class={{concat "number-format-selector__radio-" (dasherize predefinedFormat.name)}}>
+          <Radio.Option
+            @name="format"
+            @value={{predefinedFormat.format}}
+            @checked={{and (eq this.format predefinedFormat.format) (not this.isCustomFormat)}}
```

Comment:
Do we need the `(not this.isCustomFormat)`? Either the format matches or doesn't correct?